sap-s4hana-soap/2.1/modules/ROOT/pages/sap-s4hana-soap-connector-reference.adoc

```diff
@@ -0,0 +1,532 @@
+| HTTP Listener a| String |  Reference to the global HTTP Listener that will be used to receive SOAP S/4HANA messages. |  | *x*
+| WSDL Paths a| Array of <<WsdlPath>> |  Specify the locations of the WSDL definitions. |  | 
+| Name a| String |  The identifier of this element and used by other components to reference it |  | *x*
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the instance expires at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  | 
+|===
+
+==== Associated Sources
+* <<OutboundMessageListener>> 
+
+
+== Supported Operations
+
+* <<Invoke>>
+* <<Unauthorize>>
+
+[[Invoke]]
+== Invoke
+`<s4hana-soap:invoke>`
+
+Consumes an operation from an S/4HANA SOAP Web Service.
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use | | x
+| Config Ref a| ConfigurationProvider | The name of the configuration to use to execute this component |  | x
+| Service name a| String |  Name of the service |  | x
+| Operation name a| String |  The operation name of the service |  | x
+| Message a| Binary |  The content of the message. |  #[payload] |
+| Connection timeout a| Number |  Connection timeout for the request |  |
+| Connection Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit to use in the Timeout configurations |  |
+| Custom HTTP headers a| Object |  Custom headers that the request will include. The headers specified here will be merged with the default headers specified in the configuration. |  |
+| Custom query parameters a| Object | Custom query parameters that the request will include. The query parameters specified here will be merged with the default query parameters specified in the config. |  |
+| Target Variable a| String |  The name of a variable in which to store the operation's output. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  #[payload] |
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  A retry strategy in case of connectivity errors |  |
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |<<SoapOutputEnvelope>>
+| Attributes Type a| <<SoapAttributes>>
+|===
+
+=== For Configurations
+
+* <<Config>>
+
+=== Throws
+
+* S4HANA-SOAP:CONNECTIVITY
+* S4HANA-SOAP:PROCESSING_ERROR
+* S4HANA-SOAP:RETRY_EXHAUSTED
+
+[[Unauthorize]]
+== Unauthorize
+`<s4hana-soap:unauthorize>`
+
+Deletes all the access token information of a given resource owner ID so that it's impossible to execute any operation for that user without repeating the authorization dance.
```

Comment:
A nit, but I think "all the" is not correct here. It should be "all of the" or just "the."